apps/extension/src/routes/popup/approval/origin.tsx

```diff
@@ -89,24 +88,13 @@ export const OriginApproval = () => {
                 </div>
               </div>
             </div>
-            <div className='mt-3 flex flex-col gap-3'>
-              <div className='text-center text-muted-foreground'>
-                This host wants to connect to your wallet.
-              </div>
-              <div className='flex items-center gap-2 text-rust'>
-                <ExclamationTriangleIcon />
-                Approval will allow this host to see your balance and transaction history.
-              </div>
+            <div className='mt-3 flex flex-col gap-3 text-muted-foreground'>
+              <p>This host wants to connect to your wallet.</p>
+              <p>Approval will allow this host to see your balance and transaction history.</p>
```

Comment:
I rather prefer keeping the orange approval dialogue and warning symbol, but we should increase the brightness and use darker color / tint